conduit/pipeline.go

```diff
@@ -140,10 +140,12 @@ type pipelineImpl struct {
 
 	initProvider *data.InitProvider
 
-	importer   *importers.Importer
-	processors []*processors.Processor
-	exporter   *exporters.Exporter
-	round      basics.Round
+	importer         *importers.Importer
+	processors       []*processors.Processor
+	exporter         *exporters.Exporter
+	completeCallback []Completed
```

Comment:
I'm not sure if this needs to be `[]*Completed` or not. I think all of our plugins use pointer receivers, so they are already pointers, but maybe that isn't always the case.